lib/Dialect/FIRRTL/Transforms/LowerTypes.cpp

```diff
@@ -97,6 +97,7 @@ static bool hasZeroBitWidth(FIRRTLType type) {
       .Case<FIRRTLBaseType>([](auto groundType) {
         return firrtl::getBitWidth(groundType).value_or(0) == 0;
       })
+      .Case<RefType>([](auto) { return false; })
```

Comment:
Is this disallowed by a verifier? I actually think we want to allow this as users may GCT tap or view a zero-width type or a type which is inferred to be zero width.

Reply:
Oh, sure thing.  Thanks!